profiles/serializers.py

```diff
@@ -227,3 +260,32 @@ class Meta:  # pylint: disable=missing-docstring
             'edx_level_of_education',
             'education'
         )
+
+
+class ProfileFilledOutSerializer(ProfileSerializer):
+    """Serializer for Profile objects which require filled_out = True"""
+    work_history = EmploymentFilledOutSerializer(many=True)
+    education = EducationFilledOutSerializer(many=True)
+
+    def __init__(self, *args, **kwargs):
+        """
+        Update serializer_field_mapping to use fields setting required=True
+        """
+        for field in self.fields.values():
+            field.required = True
+            field.allow_null = False
+            field.allow_blank = False
```

Comment:
we really should put this loop into a separate method. it's repeated 3 times. this method could do the trick.

```python
def set_fields_to_required(self, ignore_fields=None):
    if ignore_fields:
        fields_to_require = list(field for name, field in serializer.fields.items() if name not in ignore_fields)
    else:
        fields_to_require = serializer.fields.values()
    for field in fields_to_require:
        field.required = True
        field.allow_null = False
        field.allow_blank = False
```

one option would be to add a mixin with this function and inherit that in each of these filled out serializers